radar/2024-10-1/bulcoin.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Blockchain State: Chuvashia - Volga Bulgaria"
+ring: develop
+quadrant: governance
```

Comment:
The radar was created quite recently, and not all processes have been established yet. That’s why your project is very useful — it will help us define and formulate clear and understandable criteria. The team plans to gather and discuss this. In particular, there’s a question about which project should be considered `governance` and which is still `community`. This will be discussed in the near future, and I’ll share the results afterward